canals/pipeline/pipeline.py

```diff
@@ -337,6 +341,18 @@ def get_component(self, name: str) -> Component:
         except KeyError as exc:
             raise ValueError(f"Component named {name} not found in the pipeline.") from exc
 
+    def inputs(self):
+        """
+        Returns a dictionary with the input names and types that this pipeline accepts.
+        """
+        return _describe_pipeline_inputs(self.graph)
```

Comment:
This indirection is not necessary.

Reply:
Do you prefer not having the method at all in favor of `describe_pipeline_inputs(pipeline)`, or moving the content of the function in here?

Reply:
I would move the logic in here.